assets/concepts/NutsOrganizationCredential.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
sinds it is a template, rename the file to `NutsOrganizationCredential.json.template`?